lib/src/subscription.dart

```diff
@@ -272,21 +273,22 @@ class MutableSubscriptionSet extends SubscriptionSet {
     return Subscription._(realmCore.insertOrAssignSubscription(this, query, name, update));
   }
 
-  /// Remove the [subscription] from the set, if it exists.
+  /// Removes the [subscription] from the set, if it exists.
```

Comment:
Why change the tense? If we do, there are doc comments on other methods like `clear` that should change as well..